ui/src/main/webapp/src/app/analysis-context/select-applications.component.ts

```diff
@@ -0,0 +1,51 @@
+import {Component, Input, EventEmitter, Output} from "@angular/core";
+import {RegisteredApplication} from "windup-services";
+
+@Component({
+    selector: 'wu-select-applications',
+    templateUrl: './select-applications.component.html'
+})
+export class SelectApplicationsComponent {
+    @Input()
+    availableApps:RegisteredApplication[];
+    _selectedApps:RegisteredApplication[];
+
+    @Output()
+    selectedAppsChange:EventEmitter<RegisteredApplication[]> = new EventEmitter<RegisteredApplication[]>();
+
+    @Input()
+    set selectedApps(apps:RegisteredApplication[]) {
+        this._selectedApps = apps;
+        this.selectedAppsChange.emit(this._selectedApps);
```

Comment:
I'm not totally sure that I understand what you mean. It isn't changing anything other than the property being @Input() is it?